src/EPR.RegulatorService.Frontend.Core/Services/MockedFacadeService.cs

```diff
@@ -347,4 +348,5 @@ private static List<Registration> GenerateRegulatorRegistrations()
     }
 
     public async Task<EndpointResponseStatus> AddRemoveApprovedUser(AddRemoveApprovedUserRequest request) => await Task.FromResult(EndpointResponseStatus.Success);
+    public Task<HttpResponseMessage> GetFileDownload(FileDownloadRequest request) => throw new NotImplementedException();
```

Comment:
Do we need a mocked file download function for testing?

Reply:
Its been added